scss/main.scss

```
@@ -7,5 +9,52 @@
 .container {
   display: flex;
   flex-direction: column;
-  padding: 71px 20px 0 20px;
+}
+
+.header-container {
+  display: flex;
+  flex-direction: column;
+  padding: 71px 15px 38px 15px;
+
+  & .header-title {
+    color: $primary-color;
+    font-family: $font-tittle;
+    font-size: 36px;
+    font-weight: 700;
+    letter-spacing: -0.72px;
+    margin-bottom: 38px;
+  }
+
+  & .container-crew {
+    display: flex;
+    flex-direction: column;
+
+    & .team-title {
+      color: $primary-color;
+      font-family: $font-description;
+      font-size: 18px;
+      margin-bottom: 10px;
+      text-transform: uppercase;
+    }
+
+    & .team-description {
+      color: $primary-color;
+      font-family: $font-description;
+      font-size: 14px;
+      line-height: 20px;
+      width: 348px;
+    }
+  }
+}
+
+@media (min-width: 960px) {
+  .header-container {
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 110px 243px 96px 243px;
```

I don't like these numbers... they seem so arbitrary... 🤔

---

How can I put them? for me was the best way to make it look like the original design 😿

---

you need to check what's fit better for your code, in that case imagine the designer strugling to put those numbers, I would prefer:
```suggestion
    padding: 100px 240px;
```
but